lib/common-types.ts

```diff
@@ -740,6 +741,10 @@ export type MRActions = {
       payload: CreateEntryProps<any>
       return: EntryProps<any>
     }
+    references: {
+      params: GetSpaceEnvironmentParams & { entryId: string; maxDepth: number }
```

Comment:
you might have to make it a default here too:

```suggestion
      params: GetSpaceEnvironmentParams & { entryId: string; maxDepth?: number }
```